Update documentation of KNX for dependency bump (#14567)
home-assistant/home-assistant.github.io: source/_integrations/knx.markdown

  required: false
  default: 20
  type: integer
individual_address:

>   description: The individual address.

Reviewer: ```suggestion
  description: The KNX individual address that shall be used. Note: If tunnelling server assigns an IA for a connection this will be used instead.
```

>   description: The KNX individual address that shall be used. Note: If tunnelling server assigns an IA for a connection this will be used instead.

Reviewer: ```suggestion
  description: The KNX individual address that shall be used for routing or if a tunnelling server doesn't assign an IA at connection.
```
":" is not allowed here, sorry.

  required: false
  type: string
  default: "15.15.250"
multicast_group:

>   description: The multicast group to use for automatically scanning your network.

Reviewer: ```suggestion
  description: The multicast group to use for automatic interface discovery and routing communication.
```

  required: false
  type: string
  default: "224.0.23.12"
multicast_port:

>   description: The port for multicast discovery.

Reviewer: ```suggestion
  description: The port for multicast communication.
```
